test/HappyPheanstalkDriverTest.php

```diff
@@ -111,9 +112,9 @@ public function testJobsAreBuriedWithRetry()
 
     public function testFailWithADeleteFailureStrategyRemovesTheJob()
     {
-        $driver = new PheanstalkDriver($this->conn, $this->serializer, [
+        $driver = new PheanstalkDriver($this->conn, $this->serializer, new ArrayOptions([
             'reserve-timeout' => 1,
```

Comment:
should this use the ArrayOptions::RESERVE_TIMEOUT constant like below?

Reply:
or should this just use $this->driver instead of setting up a new one?

Reply:
It should not use `$this->driver` as it's constructing the driver a different way than what's in `setUp`.